assets/src/edit-story/components/popup/index.js

```diff
@@ -29,9 +34,37 @@ const Container = styled.div.attrs(({ x, y }) => ({
 }))`
   position: fixed;
   z-index: 2147483646;
-  width: ${({ width }) => width}px;
+  width: ${({ width }) => (width === 'auto' ? width : `${width}px`)};
   max-height: ${MAX_HEIGHT}px;
   overflow: auto;
+
+  /*
+   * Custom dark scrollbars for Chromium & Firefox.
```

Comment:
Don't want to bikeshed this here, but imho the dark scrollbars look a bit wrong to me here. I like all other customizations for them though.

Reply:
Created gray style scrollbar. I think at least we need to have same size and the scrollbar format. So instead of remove it used gray style scrollbar.